Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -128,4 +143,29 @@ export class AppComponent implements OnInit {
       this.router.navigateByUrl('/');
     });
   }
+  public openSnackBar() {
```

Comment:
I would just rename the function to something more descriptive. `openUrlSnackbar` or something like